sql/2019/07_Performance/07_15.sql

```diff
@@ -0,0 +1,16 @@
+#standardSQL
+# 07_15: Percentiles of first cpu idle
+#This metric comes from Lighthouse
+SELECT
+   _TABLE_SUFFIX AS client,
+  ROUND(APPROX_QUANTILES(CAST(IFNULL(JSON_EXTRACT(report, "$.audits.first-interactive.numericValue"), JSON_EXTRACT(report, "$.audits.first-cpu-idle.numericValue")) AS FLOAT64), 1000)[OFFSET(100)] / 1000, 2) AS p10,

```

Comment:
Should this just query the first-cpu-idle audit directly? Why do you check first-interactive first? cc @patrickhulce 

Also, I'd suggest doing the JSON extraction in a nested SELECT, similar to 07_13. That way you only extract once per URL as opposed to once per percentile per URL.

Reply:
Fixed. Moved JSON extraction in a nested SELECT.